utils.py

```diff
@@ -39,6 +39,7 @@ def send_email(logger, subject, msg, recipient=None, attachment=None):
         recipient = app.config["ADMIN_EMAIL"]
     from_email = From(app.config["FROM_EMAIL"], app.config["FROM_NAME"])
     template_path = "templates/email_template.html"
+    # pylint: disable=unspecified-encoding
```

Comment:
add `encoding: 'utf-8'` dont just ignore the pylint

Reply:
somethings changed in pylint.... its erroring when it wasnt erroring before.

Reply:
added encoding='utf-8'